src/patternfly/components/DataList/data-list.scss

```diff
@@ -0,0 +1,207 @@
+@import "../../patternfly-utilities";
+
+.pf-c-data-list {
+  --pf-c-data-list--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-data-list--BoxShadow: var(--pf-global--BoxShadow--md);
```

Comment:
If I remember correctly, `pf-c-data-list` would not have box shadow, rather the `pf-c-card` would provide that.

Reply:
updated